packages/swap/src/utils/disallowChannel.ts

```diff
@@ -31,7 +31,8 @@ const isTooRisky = async (address: string | undefined): Promise<boolean> => {
 
     return parsed.risk_score !== null && parsed.risk_score >= env.ELLIPTIC_RISK_SCORE_TOLERANCE;
   } catch (error) {
-    logger.error('failed to request risk score from elliptic', { error });
+    const level = isAxiosError(error) && error.status === 404 ? 'warn' : 'error';
+    logger[level]('failed to request risk score from elliptic', { error });
```

Comment:
```suggestion
    logger[level]('failed to request risk score from elliptic', { raw: error });
```
I think having the word `error` makes it still count as error on grafana

Reply:
it now checks for `level=error`